### DISCUSSION TITLE
Store aws/gcs url for images instead of public url

### DISCUSSION THREAD

**Discussion by mayankpunetha007** [directus/directus#10606] [Archive]
Hi,

Is there a way to store the s3/gcs URL instead of the public_url when storing a rich text input. For example:

While adding a rich text I am storing the following html with images:

```
<p><img src="http://localhost:8055/assets/1d13b92c-f800-4ca4-8eec-490e0c1bc91d?width=1024&amp;height=156" alt="Faf476f4a0af09a2d8a89a82c854876d" />We will be meeting to decide the weekly Goals.</p>
<p>This event is restricted to members.<br /><br /></p>
```

Can I make this a s3/gcs or an endpoint something like:

```
<p><img src="https://storage.googleapis.com/bucketname/images/1d13b92c-f800-4ca4-8eec-490e0c1bc91d.png/>We will be meeting to decide the weekly Goals.</p>
<p>This event is restricted to members.<br /><br /></p>
```

This saves a lot of overhead to the system, By going to the public cloud URL instead of the server.

Thanks in advance!

**Comment by azrikahar** [comment]
There's an ongoing discussion exactly for this over at #7612 👍 Feel free to chime in over there so we can consolidate discussions of the same topic easier.
